qt/widgets/instrumentview/src/InstrumentWidget.cpp

```diff
@@ -264,9 +264,9 @@ void InstrumentWidget::init(bool resetGeometry, bool autoscaling,
       QString defaultView =
           QString::fromStdString(m_instrumentActor->getDefaultView());
       if (defaultView == "3D" &&
-          Mantid::Kernel::ConfigService::Instance()
-              .getValue<bool>("MantidOptions.InstrumentView.UseOpenGL")
-              .get_value_or(false)) {
+          !Mantid::Kernel::ConfigService::Instance()
```

Comment:
Why this change? Isn't it essentially the same thing? Were you getting errors with the stored value not evaluation to false?